### ISSUE TITLE
Provide better (more) maps

### ISSUE THREAD

**Issue by JAnDud** [commons-app/apps-android-commons#4918]
### What is the user problem or growth opportunity you want to see solved?

as user of OSM mapnik and [mapy.cz](http://mapy.cz/) I found used map too much googlish (only streets and rivers, missing many ponds, forests, landmarks etc). It should be fine if I can select between more maps (see c-geo app for geocaching)

![3xDoudleby](https://user-images.githubusercontent.com/19319822/159804954-ddae4eca-cbf9-4341-a9eb-f9db3055f031.jpg)
WMF Map, mapy.cz, openstreetmap.org
Find cross in the middle of field.
https://commons.wikimedia.org/wiki/File:3xDoudleby.jpg

**Comment by nicolas-raoul** [comment]
We changed maps providers, is it better now?

**Comment by JAnDud** [comment]
No, there are still only WMF maps, minor roads between fields are hard to recognize, landmarks are misssing, public buildings are not recognizeable

**Comment by nicolas-raoul** [comment]
We use WMF maps. Does anyone know whether WMF offer different map styles, rather than just one? If not, the issue probably needs to be raised on Wikimedia's phabricator.

**Comment by JAnDud** [comment]
https://phabricator.wikimedia.org/T280460
